.github/workflows/twister.yaml

```diff
@@ -206,6 +206,11 @@ jobs:
           echo "github.base_ref: ${{ github.base_ref }}"
           echo "github.ref_name: ${{ github.ref_name }}"
 
+      - name: Check MbedTLS auto-generated files
+        run: |
+          echo "Check MbedTLS auto-generated files"
+          ./modules/mbedtls/create_psa_files.py --check
+
```

Comment:
This should not really be in twister.yaml, it could go in a different workflow but since it's implemented as a check option rather than an actualy pytest I don't there's a good place to put it.

How about instead of trying to run it directly from a workflow you add the check to one of the existing test or samples? (like the other example I pointed out in my comment). Maybe this could live as part of the tfm_integration test

Reply:
>  it could go in a different workflow

Initially I added this check in `scripts_test.yaml` as you suggested, but that workfile is executed on the Zephyr repo itself while my script requires the MbedTLS repo to be present as well in order to work properly (i.e. `west init` must be issued). That's why I moved it from `scripts_test.yaml` to `twister.yaml`

> How about instead of trying to run it directly from a workflow you add the check to one of the existing test or samples?

I will try this option

Reply:
Turns out that I overlooked the 2nd option you wrote [previously](https://github.com/zephyrproject-rtos/zephyr/pull/72243#issuecomment-2111907143), but it was actually the easiest way to achieve what I was meant to do here. This should now be fixed in https://github.com/zephyrproject-rtos/zephyr/commit/37769f4f74fc5754a4e8f75abd9131eea68ed8cb.
Thanks again!